docs/docsite/rst/userguide/inventories.rst

```diff
@@ -1216,10 +1261,12 @@ The Run command window opens.
 |ad hoc-commands-inventory-run-command|
 
 .. |ad hoc-commands-inventory-run-command| image:: ../common/images/ad-hoc-commands-inventory-run-command.png
+   :alt: ad hoc commands inventory run command
 
 4. Click **Next** to choose the |ee| you want the ad-hoc command to be run against.
 
 .. image:: ../common/images/ad-hoc-commands-inventory-run-command-ee.png
+   :alt: ad hoc inventory run command ee
```

Comment:
Suggested wording: "Ad Hoc Run command dialog prompting for Execution Environments with two listed to choose from"